blkid/probe_unix.go

```diff
@@ -0,0 +1,49 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+//go:build unix && !linux
```

Comment:
let's call it `probe_other.go` to make it less confusing, as `_unix` automatically injects a build tag